packages/framework-provider-azure-infrastructure/src/infrastructure/types/functionDefinition.ts

```diff
@@ -0,0 +1,37 @@
+export interface IBinding {
+  type: string
+  name: string
+  direction: string
+}
+
+export interface IScheduleBinding extends IBinding {
+  schedule: string
+}
```

Comment:
It's better use the following style, as it's used throughout the project:

```suggestion
export type IScheduleBinding = IBinding & {
  schedule: string
}
```

Reply:
Cool, thanks